### DISCUSSION TITLE
NEW COMPONENT<title>

### DISCUSSION THREAD

**Discussion by leah-77** [community/community#49914] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello, I'm starting to work with Angular. I created a new component, but I can't use it. Its content is not displayed in the browser. Another thing, I work with the software: StackBlitz and the Angular files it creates do not have a component with APP files.  according to a training video I understood that I should combine there the tag according to the name of the SELECTOR created' to view the html files content.

**Comment by maahaMaya** [comment, marked as answer]
you can follow these to learn : [app.component.html](https://github.com/maahaMaya/AngularPratice)

you can declare component name in app.component.html
https://github.com/maahaMaya/AngularPratice/blob/main/myAppAngular/src/app/app.component.html

**Reply by leah-77** [reply]
i'm steal not able to create a new component in the StackBlitz.
i tryied to add an app component that doesn't appear here in StackBlitz, and appers in the instruction videos i saw,
but i get a message that there is no app component in the app.


Error in /turbo_modules/@angular/core@15.1.2/fesm2015/core.mjs (8633:15)
NG0906: The App is not an Angular component, make sure it has the `@Component` decorator.

**Comment by maahaMaya** [comment]
12sourav12s@gmail.com  

you mail me on this email
we will connect you and resolve the doubt.

**Reply by leah-77** [reply]
Thanks, I found the solution to the problem

**Comment by sadok-dorbez** [comment]
Can you share the solution please?
